test/case/infix_containers/container_bridge/test.py

```diff
@@ -84,18 +84,18 @@
             }
         })
 
-    with test.step(f"Verify {NAME} container has started"):
+    with test.step("Verify container 'web-docker0' has started"):
         c = infamy.Container(target)
         until(lambda: c.running(NAME), attempts=10)
 
-    with test.step(f"Verify {NAME} container responds"):
-        _, hport = env.ltop.xlate("host", "data")
-        url = infamy.Furl(URL)
+    _, hport = env.ltop.xlate("host", "data")
+    url = infamy.Furl(URL)
 
-        with infamy.IsolatedMacVlan(hport) as ns:
-            ns.addip(OURIP)
+    with infamy.IsolatedMacVlan(hport) as ns:
+        ns.addip(OURIP)
+        with test.step("Verify host:data can ping 10.0.0.2"):
             ns.must_reach(DUTIP)
-
+        with test.step("Verify container 'web-docker0' is reachable on http://10.0.0.2:91"):
```

Comment:
Wrong port, "Verify container web server 'web-docker0' is reachable on http://10.0.0.2:8080"

Also, maybe we should mention somewhere this is a port forward test?

Reply:
Sorry, embarrassing mistake, will fix that.